wrims-core/src/main/java/wrimsv2/components/ControlData.java

```diff
@@ -1,227 +1,7 @@
-	public static String xaSort = null;
-	public static boolean resimDate=false;
-	public static boolean resimGroundwater=false;
-	public static int resimYear;
-	public static int resimMonth;
-	public static int resimDay;
-	public static int cycleDataStartYear;
-	public static int cycleDataStartMonth;
-	public static int cycleDataStartDay;
-	    
-	public static String USER;       
-	public static String PASS;  
-	public static String databaseURL;
-	public static String sqlGroup="calsim";
-	public static int ovOption=0;
-	public static String ovFile="";
-	public static Map<String, String> ovPartBC=new HashMap<String, String>();
-	public static boolean isSimOutput = true;
-	
-	public static boolean enableProgressLog = false;
-	
-	public static LocalDateTime ldt = LocalDateTime.now();
-	public static String dateTimeAppend = DateTimeFormatter.ofPattern("yyyy-MM-dd-HH-mm", Locale.ENGLISH).format(ldt);
-
-	public static boolean printGWFuncCalls=false;
-	
-	public static int yearOutputSection=-1;
-	public static int monMemSection=-1;
-	public static int outputYear;
-	public static int outputMonth;
-	public static int outputDay;
-	public static int memStartYear;
-	public static int memStartMonth;
-	public static int memStartDay;
-	public static int prevMemYear;
-	public static int prevMemMonth;
-	public static int prevMemDay;
-	public static int prevOutputYear;
-	public static int prevOutputMonth;
-	public static int prevOutputDay;
-	public static Date prevMemDate;
-	public static Date memStartDate;
-	public static Date prevOutputDate;
-	public static int nThreads=1;
-	public static boolean unchangeGWRestart=false;
-	public static boolean genSVCatalog=true;
-	public static boolean showTimeUsage=true;
-	public static int t_cbc=0;
-	public static int t_xa=0;
-	public static int t_ts=0;
-	public static int t_svar=0;
-	public static int t_dvar=0;
-	public static int t_goal=0;
-	public static int t_wt=0;
-	public static int t_wtss=0;
-	public static int t_as=0;
-	public static int t_readTs=0;
-	public static int t_writeDss=0;
-	public static int t_cam=0;
-	public static int t_parse=0;
-	public static int t_ann=0;
-	public static int n_ann=0;
-	public static int t_annx2=0;
-	public static int n_annx2=0;
-	public static int t_annec=0;
-	public static int n_annec=0;
-	public static int t_annlinegen=0;
-	public static int n_annlinegen=0;
-	public static int t_anngetndo_x2=0;
-	public static int n_anngetndo_x2=0;
-	public static int t_anngetndo_x2_curmonndosplit=0;
-	public static int n_anngetndo_x2_curmonndosplit=0;
-	public static int t_annec_matchdsm2=0;
-	public static int n_annec_matchdsm2=0;
-	
-	public static int pid=-1;
-	
-	public ControlData(){
-	}
-	
-	public static String getPartE(){
-		return timeStep;
-	}
+@Deprecated
+public class ControlData extends gov.ca.water.wrims.engine.core.components.ControlData {
+	// Deprecated for move to new package. Retained old package location for compatibility with CalLight code
```

Comment:
correct spelling: CalLite 
Instead of a comment this could be a Javadoc calling out gov.ca.water.wrims.engine